README.md

```diff
@@ -23,9 +23,9 @@ To make NUR accessible for your login user, add the following to `~/.config/nixp
 ```nix
 {
   packageOverrides = pkgs: {
-    nur = pkgs.callPackage (import (builtins.fetchGit {
-      url = "https://github.com/nix-community/NUR";
-    })) {};
+    nur = import (builtins.fetchGit "https://github.com/nix-community/NUR") {
```

Comment:
I think it is also time to switch to `fetchTarball`, which is apparently faster then `fetchgit` - even for incremental updates - since the repository is rather small.

Reply:
Yeah, will be a bit longer though, but I think that's okay